.github/workflows/build.yaml

```diff
@@ -34,3 +34,32 @@ jobs:
           tags: |
             harbor.disembark.dev/7tv/event-api:${{ github.sha }}
             harbor.disembark.dev/7tv/event-api:latest
+
+  deploy:
+    name: Deploy (Production)
+    runs-on: ubuntu-latest
```

Comment:
Running on shared runners - would this cause issues with the new internal harbor thing?

Reply:
no its only a k8s deploy